### ISSUE TITLE
Bootstrap predicting same output for all inputs

### ISSUE THREAD

**Issue by jonoNel** [emilwallner/Screenshot-to-code#57]
So I'm using the datasets from the paper with bootstrap_generator and I have a loss of 0.1212 but the model predicts the same output for every input even on the training data, should I do more epochs? At what stage does it predict different outputs?

**Comment by jonoNel** [comment]
I increased the strides to 3 to reduce the size of the parameters, would that be the cause?
